modules/gapi/src/api/gkernel.cpp

```diff
@@ -66,6 +66,9 @@ cv::gapi::GKernelPackage cv::gapi::combine(const GKernelPackage  &lhs,
                 result.m_id_kernels.emplace(kernel.first, kernel.second);
             }
         }
+        for (const auto &transforms : lhs.m_transformations){
+            result.m_transformations.push_back(transforms);
```

Comment:
no collision detection  for transformations, is this a good thing? might be good point to scrub the vector with `std::unique` or something

Reply:
It would be tough to write all operator in a proper way, so maybe it's better to pick the easiest option considering our actual use case.

Reply:
What is collision for transformations? Having multiple transformations of the same type in the Kernel package? This can be solved by `all_unique` thing recently written by @TolyaTalamanov

Reply:
This is already done this way (that's why we need `API` defined in transformations)

Reply:
hmm, I believe there might be a corner case:
```c++
auto pkg = cv::gapi::kernels<TransformA>(); // all_unique<> passes the check
pkg.include<TransformA>(); // all_unique<> is not called at all!
```
I am not sure how to handle this, and we also can reach the same with `combine(...)`.
Let's file a bug and fix it later, there's no clear solution to this as I see it.

Reply:
Still, what's wrong with having multiple similar transforms in the package? One of those duplicates will be called anyway. If there's nothing to transform, neither of them is called (also anyway). Not perfect, but not that awful too.